From 06dfededed954e6d98450ae8f57d561359f2430e Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Sat, 12 Jan 2019 12:28:55 +0000
Subject: [PATCH] man: tc-taprio.8: fix syntax error

.Q does not exist so groff complains and the "queues" word is actually
not displayed.

Fixes: 579acb4bc52f ("taprio: Add manpage for tc-taprio(8)")

Signed-off-by: Luca Boccassi <bluca@debian.org>
Signed-off-by: Stephen Hemminger <stephen@networkplumber.org>
(cherry picked from commit 8242808ced032d56d690ae66a93fa52e8d0ada9a)
Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 man/man8/tc-taprio.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man8/tc-taprio.8 b/man/man8/tc-taprio.8
index 92055b434b942..850be9b036491 100644
--- a/man/man8/tc-taprio.8
+++ b/man/man8/tc-taprio.8
@@ -45,7 +45,7 @@ How traffic is mapped to different hardware queues is similar to
 and so the
 .B map
 and
-.Q queues
+.B queues
 parameters have the same meaning.
 
 The other parameters specify the schedule, and at what point in time
-- 
2.20.1

